plugins/GSdx/GSSetting.cpp

```diff
@@ -46,6 +46,7 @@ const char* dialog_message(int ID, bool* updateText) {
 #endif
 		case IDC_CRC_LEVEL:
 			return "Control the number of Auto-CRC hacks applied to games.\n\n"
+				"Automatic:\nAutomatically select 'Partial' or 'Full' level based on the selected renderer.\n\n"
```

Comment:
I'd prefer:
```
Automatically sets the recommended CRC hack level based on the selected renderer. 
This is the recommended setting.
Partial will be selected for OpenGL.
Full will be selected for Direct3D.
```

You can also remove the recommended at the Full/Partial levels of the tooltip, as it is now printed in the combobox for both options. And replace `Accurate/depth options may be required` with `Blending Unit Accuracy/Depth Emulation may be need to be enabled` at OpenGL.